pyaerocom/colocation/colocator.py

```diff
@@ -378,9 +380,21 @@ def run(self, var_list: list = None):
                 coldata = self._run_helper(
                     mod_var, obs_var
                 )  # note this can be ColocatedData or ColocatedDataLists
-                if mod_var not in data_out:
-                    data_out[mod_var] = {}
                 data_out[mod_var][obs_var] = coldata
+
+                if obs_var in ["conco3", "vmro3"]:
```

Comment:
We may also need this for CO. One way to do this would be to define a global variable `MDA8_VARS` which contains these variables and others we foresee needing, and check to see if obs_var is contained in that.